logic/scorer/scorer.go

```diff
@@ -108,8 +116,99 @@ func howManyAddUpTo(goal int, ptVals []int) uint {
 	return many
 }
 
-// Assumes input is sorted and has len 5
-func scoreRunsAndPairs(values []int) (scoreType, int) { //nolint:gocyclo
+func scorePairs(values [numCardsToScore]int, valuesToCounts valueToCount) (scoreType, int) {
+	pairPoints := 0
+	pairType := none
```

Comment:
:cactus: could we use the zero value to simplify some of this functions logic?

```go
var points int
var st scoreType
// for & switch
if st == 0 { // or points == 0
  return none, 0
}
return st, points

Reply:
Actually, the more that I think about this: I don't think `scorePairs` _nor_ `scoreRuns` should _ever_ return a `none`: they can only possibly claim to have knowledge about a portion of the score, so they can't say that the hand has "scored none".

Reply:
Now that you bring this up, I think `none` is essentially _never_ useful; I will also clean up uses in `master` :stare: 😉 
https://github.com/joshprzybyszewski/cribbage/blob/ca366c39cfe2d68e6985e07d188a530cf81a7399/logic/scorer/scorer.go#L67